pkg/management/postgres/metrics/collector.go

```diff
@@ -22,11 +22,11 @@ import (
 	"context"
 	"database/sql"
 	"fmt"
-	"path"
-	"regexp"
-
 	"github.com/blang/semver"
 	"github.com/prometheus/client_golang/prometheus"
+	"path"
+	"regexp"
+	"strings"
 
 	"github.com/cloudnative-pg/cloudnative-pg/pkg/management/log"
 	"github.com/cloudnative-pg/cloudnative-pg/pkg/management/postgres"
```

Comment:
Please run the linter with `make checks`. This hunk causes the linter to fail due to the import order.